serving/docs/lmi/deployment_guide/benchmarking-your-endpoint.md

```diff
@@ -111,7 +111,7 @@ TOKENIZER=<tokenizer_id> ./awscurl -c 10 -N 30 -X POST -n sagemaker https://runt
 ### Usage with SageMaker Streaming Response
 
 You can also use `awscurl` to invoke your endpoint with streaming responses. 
-This assumes that you have configured your deployment configuration with `option.output_formatter=jsonlines` so that LMI streams responses.
+This assumes that you have configured your deployment [configuration](https://github.com/deepjavalibrary/djl-serving/blob/master/serving/docs/lmi/deployment_guide/configurations.md#lmi-common-configurations) with `option.output_formatter=jsonlines` so that LMI streams responses.
```

Comment:
can we also add: `or you have passed 'stream': true as part of the payload`? I think that can cover the full set of possibilities for streaming